### PR TITLE
docs: add offer subscription

### PR DESCRIPTION
**Pull request by jjeroch**
## Description

- added app subscription activation image
- create Service Subscription Activation (Provider) md-file
- update file names
- updated description of 01. Service Subscription.md
- created 02. Service Subscription Activation (Provider).md
- created 02. App Subscription Activation (Provider).md
- created 01.App Subscription Request Management.md
- updated 02. App Subscription Activation (Provider).md
- created 01. Subscription Flow.md
- updated hyperlinks and file intro
- updated 01.App Subscription Request Management.md
- updated 03. Subscription Overview (App Provider).md
- app release process files updated and restructured
- rename 01. Intro.md to index.md
- updated app & service release index.md
- offer autosetup documentation enhanced by new endpoint with process woorker

## Why

New feature development as well as feature enhancement in backend and frontend code

## Issue

n/a

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/04. App(s)/02. App Release Process/02. App Detail Page.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<br>
+
+<p align="center">
+<img width="750" alt="image" src="https://user-images.githubusercontent.com/94133633/215898791-b832d31c-8c37-4483-af45-53b263c649c4.png">
+</p>
+
+<br>
+
+###### App Details Images
+add sume details to the function
```
Comment (evegufy) [inline docs/04. App(s)/02. App Release Process/02. App Detail Page.md:19]: Could you please adjust this sentence?

File: docs/04. App(s)/02. App Release Process/02. App Detail Page.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+</p>
+
+delete function:
+
+image images
+
+<br>
+
+###### App Documents
+add sume details to the function
```
Comment (evegufy) [inline docs/04. App(s)/02. App Release Process/02. App Detail Page.md:31]: Could you please adjust this sentence?

**Review by evegufy — CHANGES_REQUESTED** [review]

File: docs/04. App(s)/02. App Release Process/02. App Detail Page.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jjeroch) [reply docs/04. App(s)/02. App Release Process/02. App Detail Page.md]: done

File: docs/04. App(s)/02. App Release Process/02. App Detail Page.md (hunk 2 above)
Comment (jjeroch) [reply docs/04. App(s)/02. App Release Process/02. App Detail Page.md]: done

**Review by evegufy — APPROVED** [review]

**Pull request merged**